Automattic/jetpack
# AI Title Optimization: Add sidebar entry

Closes #36941 

## Proposed changes:

* Add `Improve Title` button entry on Jetpack Sidebar.
* Add `Title Optimization` initial render, with empty modal.

### Other information:


## Jetpack product discussion


N/A

## Does this pull request change what data or activity we track or use?


N/A

## Testing instructions:






* Make sure you have beta blocks activated.
* Go to your editor.
* Open Jetpack Sidebar and AI Assistant panel.
* You should see the `Improve Title` button (disabled).
* Write something on editor.
* It should be enabled and clicking it opens the Modal
